### ISSUE TITLE
Merge identical NUTS levels

### ISSUE THREAD

**Issue by madewild** [ec-doris/kohesio-backend#62]
When some NUTS levels are identical (e.g. for Cyprus), we could decide to merge them in the API so that it's less annoying for the user clicking on the map. But let's be cautious since this could have unfroreseen implications...

**Comment by D063520** [comment]
just a note: this should not be to merge the wikibase entities!

**Comment by madewild** [comment]
Indeed, important to keep them separate in the Wikibase to avoid infinite loops etc. :)

**Comment by D063520** [comment]
Useful scheme of Thierry
<img width="654" alt="Screenshot 2021-06-01 at 11 35 39" src="https://user-images.githubusercontent.com/10883585/120382466-f664b980-c323-11eb-8479-8859cf69ab87.png">

**Comment by D063520** [comment]
So I implemented this ... basically nuts regions that are annotated as "NUTS statistical only" https://linkedopendata.eu/wiki/Item:Q2727537
 are ignored and skipped. This is for example the case for
https://linkedopendata.eu/wiki/Item:Q241763
, a czech nuts.

I already annotated according to the table above the nuts regions that should only be statistical. So the non necessary nuts are eliminated ....

I'm really happy with my solution : ) except that now there are some holes in italy and poland but I know why .... will fix it tomorrow morning!
